### ISSUE TITLE
Planter and Pam's Weee flowers seeds

### ISSUE THREAD

**Issue by wbrown2wmb** [powercrystals/MineFactoryReloaded#388]
The latest release, MineFactoryReloaded-[1.7.10]2.8.0RC3-591 won't plant the flower seeds from Pam's weee flowers. Not stacks or single seeds

**Comment by JackTheRedCreeper** [comment]
Yeah, this is kinda annoying. I wanted to make a white roses farm to get nonstop bonemeal (I can get that by hand but its kinda annoying).
The harvester and the fertilizer dont recon the flowers neither >-<
Also it'd be cool if we could set a priority setting. For example, fertilizer will work on X before Y
